src/api/spec/models/event_subscription/find_for_event_spec.rb

```diff
@@ -88,12 +90,56 @@
 
 RSpec.describe EventSubscription::FindForEvent do
   describe '#subscribers' do
-    subject do
-      event = Event::CommentForProject.first
-      EventSubscription::FindForEvent.new(event).subscriptions
+    context 'with a request' do
```

Comment:
When comparing this test with [that](https://github.com/openSUSE/open-build-service/pull/3992/files#diff-ad9968a7b553554b3b3d013f2c2bc76eR160) one, it seems that both tests differe in the test preparation, but ensure the same outcome (in the actual test).
Shouldn't there be some difference? For example, right now it seems that the 'channel: :instant_email' attribute doesn't have any effect in the test. Which either means it's not needed or it's not tested (whatever effect it has).

Reply:
@bgeuken I fixed teh duplicated code here using a custom matcher and more let blocks.. Please have a look!